docs/application/dotnet/guides/nui/text.md

```diff
@@ -402,6 +402,25 @@ field.HorizontalAlignment = HorizontalAlignment.Begin;
 ```
 
 <a name="textField4"></a>
+### Use Input properties in TextField
+
+Input properties, such as `InputColor` and `InputPointSize`, are used when you want to set new input texts differently from the existing text settings when the new text inputs.
+
+Especially, these properties should be used with some specific conditions.
+
+For instance, they're used when `TextChanged` signal is emitted or a button is clicked.

```

Comment:
You can use these properties only when the `TextChanged` event occurs.

Reply:
`TextChanged` signal is one of signals when the user wants to use Input properties.
So, I don't think that 'only' when the `TextChanged` event occurs.

I'll change this phrase to :
For instance, you can use these properties when the `TextChanged` event occurs or when the `Clicked` event of Button occurs.